platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/dsl/KotlinDslPluginCrossVersionSmokeTest.kt

```diff
@@ -208,6 +212,9 @@ class KotlinDslPluginCrossVersionSmokeTest : AbstractKotlinIntegrationTest() {
                 "Consult the upgrading guide for further information: " +
                 "https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecated_access_to_conventions"
         )
+        if (!GradleContextualExecuter.isConfigCache()) {
```

Comment:
💅 Favor `isNotConfigCache()` over `!isConfigCache()`
```suggestion
        if (GradleContextualExecuter.isNotConfigCache()) {
```